src/windows/export.py

```diff
@@ -370,30 +381,36 @@ def cboSimpleProjectType_index_changed(self, widget, index):
         # parse the xml files and get targets that match the project type
         project_types = []
         acceleration_types = {}
-        for preset_path in [info.EXPORT_PRESETS_PATH, info.USER_PRESETS_PATH]:
-            for file in os.listdir(preset_path):
-                xmldoc = xml.parse(os.path.join(preset_path, file))
-                type = xmldoc.getElementsByTagName("type")
-
-                if _(type[0].childNodes[0].data) == selected_project:
-                    titles = xmldoc.getElementsByTagName("title")
-                    videocodecs = xmldoc.getElementsByTagName("videocodec")
-                    for title in titles:
-                        project_types.append(_(title.childNodes[0].data))
-                    for codec in videocodecs:
-                        codec_text = codec.childNodes[0].data
-                        if "vaapi" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-vaapi.svg")
-                        elif "nvenc" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-nvenc.svg")
-                        elif "dxva2" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-dx.svg")
-                        elif "videotoolbox" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-vtb.svg")
-                        elif "qsv" in codec_text and openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-qsv.svg")
-                        elif openshot.FFmpegWriter.IsValidCodec(codec_text):
-                            acceleration_types[_(title.childNodes[0].data)] = QIcon(":/hw/hw-accel-none.svg")
+        for preset_folder in [info.EXPORT_PRESETS_PATH, info.USER_PRESETS_PATH]:
+            for file in os.listdir(preset_folder):
+                preset_path = os.path.join(preset_folder, file)
+                try:
+                    xmldoc = xml.parse(preset_path)
+                    type = xmldoc.getElementsByTagName("type")
+
+                    if _(type[0].childNodes[0].data) == selected_project:
+                        titles = xmldoc.getElementsByTagName("title")
+                        videocodecs = xmldoc.getElementsByTagName("videocodec")
+                        for title in titles:
+                            project_types.append(_(title.childNodes[0].data))
```

Comment:
```suggestion
                        project_types.extend([_(t.childNodes[0].data) for t in titles])
```

Oops, only just noticed this one.  Far more Pythonic to say...

Reply:
(I realize you didn't write that line originally, but if we're touching it, might as well make it pretty.)

Reply:
This is c++ innovation that were adopted in Python, so it is not "Pythonic" in any way, while it still may look like this. This kind of code is harder to read than the original one.

Reply:
Masking `append` to `extend` behind "beautify" of the code.

Reply:
When the list is constructed by a loop regardless, I agree it's less impactful. however `.extend()` isn't just a different way of writing `.append()` in a loop. List concatenation is more efficient than repeatedly appending individual items, primarily because the memory allocation required to enlarge the destination list can be done all at once.

The larger the list you're adding to, and the larger the list of items you're extending it _with_, the more benefits this has. And while you benefit the most if the lists are preexisting, even when looping to construct the list of additions, doing that first and _then_ adding it to the destination list is preferable.

IOW, if your objection is primarily to the comprehension, then this version without it is still preferable, because it will avoid repeated small allocations to increase the size of `project_types`
```python
title_types = []
for t in titles:
    title_types.append(_(title.childNodes[0].data))
project_types.extend(title_types)
```
The comprehension is just a cleaner way of writing that concisely, though I realize not everyone cares for them.

Reply:
`listA.extend(listB)` is literally **ten times** as fast as `.append()` in a loop. And even if you loop to create the second list, doing that first and `.extend()`ing afterwards takes only half as long as the looped `.append()` on the destination list.

These timeit runs use exaggeratedly large lists, so real-world this won't have **nearly** as big of an impact, but...
```python3
# listA = a million integers from 0...999999
# listB = another million, from 2000000...2999999
#
# Looping listA.append() for each item in listB
>>> timeit.repeat('for n in listB:\n    listA.append(n)', setup='listA = [x for x in range(0, 1000000)]\nlistB = [x for x in range(2000000, 3000000)]', number=100, globals=globals())
[6.985341801017057, 7.013206199044362, 6.989441379031632, 7.000579444982577, 6.9871047679916956]

# listA.extend() with the entire contents of listB
>>> timeit.repeat('listA.extend(listB)', setup='listA = [x for x in range(0, 1000000)]\nlistB = [x for x in range(2000000, 3000000)]', number=100, globals=globals())
[0.6946912699495442, 0.6940702060237527, 0.694431746029295, 0.6942416160018183, 0.6937777320272289]

# listA.extend() with a list-comprehension copy of listB:
>>> timeit.repeat('listA.extend([n for n in listB])', setup='listA = [x for x in range(0, 1000000)]\nlistB = [x for x in range(2000000, 3000000)]', number=100, globals=globals())
[3.4050588610116392, 3.4058208650094457, 3.4050478630233556, 3.404870489030145, 3.405109421000816]
```

Reply:
Now, again, I'm **not** saying this actually _matters_ in this particular case. It doesn't. We're talking about small lists here, where it really doesn't make the tiniest bit of difference how you build them. But certain code patterns have more efficient alternatives that are typically preferable. Taking a blanket position of, "`append()` is clearer than `extend()`" ignores the fact that they're _not_ even close to equivalent in terms of performance/efficiency, because there are times when that will matter.